pkg/virt-handler/cgroup/cgroup.go

```diff
@@ -56,13 +58,34 @@ type Manager interface {
 
 	// GetCpuSet returns the cpu set
 	GetCpuSet() (string, error)
+
+	// Create new child cgroup
+	CreateChildCgroup(string, string) (bool, error)
+
+	// Attach TID to cgroup
+	AttachTID(string, string, int) error
 }
 
 // This is here so that mockgen would create a mock out of it. That way we would have a mocked runc manager.
 type runcManager interface {
 	runc_cgroups.Manager
 }
 
+// If a task is moved into a sub-cgroup, we want the manager to
+// reference the root cgroup, not the sub-cgroup.
+//
+// Currently the only sub-cgroup create is named "housekeeping".
+
+func managerPath(taskPath string) string {
+	retPath := taskPath
+	s := strings.Split(taskPath, "/")
+	if s[len(s)-1] == "housekeeping" {
+		fStr := strings.Join(s[1:len(s)-1], "/") + "/"
```

Comment:
Why do we need to drop the leading `/` here? This will transform the path from absolute to relative: `/sys/fs/cgroup/.../housekeeping` -> `sys/fs/cgroup/.../`. Also, why do we need to add a trailing `/` at the end?

Reply:
Good catch! Fixed, thanks.